modules/ROOT/pages/architecture-visualization.adoc

```diff
@@ -0,0 +1,44 @@
+= Architecture Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== About the Architecture Visualization
+
+The Architecture Visualization shows the topology of your application network. In this visualization, you can assign change display names of nodes, add tags, and assign nodes to layers. The Architecture Visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.
+
+== View Information about One Service in your Network
+
+When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are unavailable.
```

Comment:
Does your audience know what graph we are referring to?

Reply:
@asimons-mule should we say "canvas" here instead?

Reply:
Yes, canvas will work